AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -42,6 +42,11 @@ extension Target.Dependency {
     static var SmithyXML: Self { .product(name: "SmithyXML", package: "smithy-swift") }
 }
 
+extension Target.PluginUsage {
+    // Smithy plugins
+    static var SmithyCodeGenerator: Self { .plugin(name: "SmithyCodeGenerator", package: "smithy-swift") }
+}
+
```

Comment:
Above, a convenience var with the code generator plugin is defined.

Below, the plugin is inserted into all of our service clients (public and internal.)